recipes/easydict/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = "1.9" %}
+package:
+    name: easydict
+    version: {{ version }}
+source:
+    sha256: 3f3f0dab07c299f0f4df032db1f388d985bb57fa4c5be30acd25c5f9a516883b
+    url: https://pypi.io/packages/source/e/easydict/easydict-{{ version }}.tar.gz
+build:
```

Comment:
This package can probably be made `noarch: python`. See the docs here: https://conda-forge.org/docs/meta.html#building-noarch-packages